src/components/SearchForm/_search-form.scss

```diff
@@ -43,3 +47,20 @@
   color: var(--colour-white);
   margin-right: 0;
 }
+
+.search-form__btn {
+  padding-bottom: calc(2 * var(--space-unit));
+  padding-left: calc(1.5 * var(--space-unit));
+  padding-right: calc(3.125 * var(--space-unit));
+  padding-top: calc(2 * var(--space-unit));
+  position: absolute;
+  right: 0;
+
+  .btn__icon--right {
+    margin: 0;
+  }
+
+  .btn__text {
+    @extend %visually-hidden;
+  }
```

Comment:
To avoid nesting styles, I think we could add some props to the Button component to allow us to set classnames on the btn__text and btn__icon elements. I noticed we've nested these selectors (btn__icon & btn__text) elsewhere too (e.g. _accordion.scss), so I think it would help fix these issues too.

What do you think? (& @creido-welly, too!)

Reply:
Reducing nesting is definitely cleaner and keeps separation between components. The Button component already accepts className as a prop so lets use it.

Reply:
I think it may need `textClassName` and `iconClassName` too though, in this instance?

Reply:
@chriddell this is updated now :)